esp/src/eclwatch/XrefLostFilesWidget.js

```diff
@@ -89,19 +134,32 @@ define([
                 var results = [];
                 var newRows = [];
                 if (lang.exists("DFUXRefLostFilesQueryResponse.DFUXRefLostFilesQueryResult.File", response)) {
-                    results = response.DFUXRefLostFilesQueryResponse.DFUXRefLostFilesQueryResultFile
+                    results = response.DFUXRefLostFilesQueryResponse.DFUXRefLostFilesQueryResult.File
+                
+                    if (results.length) {
+                        arrayUtil.forEach(results, function (row, idx) {
+                           newRows.push({
+                                Name: row.Name,
+                                Modified: row.Modified,
+                                Numparts: row.Numparts,
+                                Size: row.Size,
+                                Partslost: row.Partslost,
+                                Primarylost: row.Primarylost,
+                                Replicatedlost: row.Replicatedlost
+                            });
+                        });
+                    } else if (results.Name) {
+                        newRows.push({
+                            Name: results.Name,
+                            Modified: results.Modified,
+                            Numparts: results.Numparts,
+                            Size: results.Size,
+                            Partslost: results.Partslost,
+                            Primarylost: results.Primarylost,
+                            Replicatedlost: results.Replicatedlost
+                        });
+                    }
```

Comment:
Note:  Having to check the response structures from ESP for the existence of an array smells like a bug in the ESP API (which may be too late to do anything about).

I would suggest moving this "cleanup" logic into WsDFUXref.